lms/djangoapps/django_comment_client/permissions.py

```diff
@@ -118,6 +119,16 @@ def test(user, per, operator="or"):
         if isinstance(per, basestring):
             if per in CONDITIONS:
                 return _check_condition(user, per, content)
+            # TODO: Consider changing this to something a little more clear
+            if 'group_' in per:
+                # If a course does not have divided discussions
+                # or a course has divided discussions, but the current user's content group does not equal
+                # the content group of the commenter/poster,
+                # then the current user does not have group edit permissions.
+                division_scheme = get_course_discussion_settings(course_id).division_scheme
+                if division_scheme is 'none' or user_group_id is None or content_user_group is None\
```

Comment:
Note: You can use parenthesis around the condition for multiple lines rather than a slash.